similarity/src/main/java/edu/illinois/cs/cogcomp/wsim/esa/ResourcesConfig.java

```diff
@@ -0,0 +1,54 @@
+package edu.illinois.cs.cogcomp.wsim.esa;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
+
+import edu.illinois.cs.cogcomp.classification.hierarchy.datastructure.StopWords;
+
+/**
+ * Shaoshi Ling
+ * sling3@illinois.edu
+ */
+

```

Comment:
please rewrite as a Configurator (https://github.com/CogComp/cogcomp-nlp/blob/master/core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/utilities/configuration/Configurator.java) unless there's a compelling reason to do otherwise.